### PR TITLE
autopublish to npm, fix wavy

### PR DESCRIPTION
**Pull request by gunar**

### CODE REVIEW

**Comment by gunar** [comment]
blocker: https://github.com/invisible-tech/chance-extras/pull/3

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "builddoc": "jsdoc -c .jsdoc.json",
     "coverage": "nyc report --reporter=lcov --reporter=html --reporter=text-summary --reporter=text",
     "fix": "eslint --cache --fix scripts src test",
-    "lint": "eslint --cache -f node_modules/eslint-formatter-pretty scripts src test",
+    "lint": "eslint --cache -f node_modules/eslint-formatter-pretty --ignore-path .gitignore .",
```
Comment (kewitz) [inline package.json]: Should we also ignore `node_modules`?

Comment (gunar) [reply package.json]: Yes, that is indirectly ignored by `.gitignore`. Got it? :-)

**Pull request merged**